changes/140.changes

```diff
@@ -0,0 +1 @@
+`recombinant_update` with `force` will now DROP old columns from the DataStore for fields that do not exist in the Schema.
```

Comment:
This seems like a good reason to not give users an interface to do this from the UI. Dropping columns will remove data that we might need as part of a missing or failed data migration.

Reply:
@wardi so should the tables just always leave old columns in them? would that not affect datastore_search?

Reply:
Oh I guess it is only when the `--force` option is set eh? so as long as that is a sysadmin only thing it should be fine?